### PR TITLE
add comment count feature

### PR DESCRIPTION
**Pull request by karobarari**

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -76,6 +77,7 @@ describe("GET /api/articles/:article_id", () => {
           created_at: expect.any(String),
           votes: expect.any(Number),
           article_img_url: expect.any(String),
+          comment_count: expect.any(String),
```
Comment (nickkagame) [inline __tests__/app.test.js]: Could you check this in a different test?

**Review by nickkagame — CHANGES_REQUESTED** [review]
Hi Karo,  Would you be able just write a separate test for comment count rather than updating the old one?

**Comment by karobarari** [comment]
Hi 
I was in the middle of hosting and commited on this branch can I delete one of the commits?

**Review by Keb84 — APPROVED** [review]
Nice work 🎸 merge when ready 🚀

**Review by nickkagame — APPROVED** [review]
Nice work!

**Pull request merged**